### ISSUE TITLE
Safe or piggy bank to store money

### ISSUE THREAD

**Issue by Eikkiyi** [StardewModders/Modding-Ideas#173]
i was thinking about this thing i'd love to see for a stardew valley mod so here i go

it'd be cool to manage my money by putting it in a piggy bank or a safe or some sort-- 

for example i'd want to save up for a barn upgrade but i just seem to keep spending all of my money on crop without realizing whoops! it'd help out a ton by putting aside a thousand or whatever amount then withdraw it if i'm ready for the upgrade?

tl;dr banking of some sort mod

**Comment by Platonymous** [comment]
Not a piggy bank but should do the trick:
https://www.nexusmods.com/stardewvalley/mods/3135
